### PR TITLE
Basic Cities front

### PR DESCRIPTION
**Pull request by Light-it-luc**
# Light-it Pull Request Template ⚡⚡⚡

## Description ⭐

Created a simple frontend to display cities and a simple CityController to handle
the city endpoints. Implemented backend pagination to display results.

### Type of change 🏃

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] This change requires a documentation update

### Requires ❗

- [ ] This change requires a database update or migration
- [x] This change requires a documentation update

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g., Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!
> 🟡 Why was the default value removed?

E.g., Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

|              |                |                                     |
| ------------ | -------------- | ----------------------------------- |
| Blocking     | 🔴 ❌ 🚨       | RED                                 |
| Non-blocking | 🟡 💡 🤔 💭    | Yellow, thinking, etc               |
| Praise       | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |

## Links

- [Git Flow](https://lightit.slite.com/app/docs/SC8usN2Ju)
- [Handbook of good practices for reviewers in Code Reviews](https://lightit.slite.com/app/docs/ddNGohWthVB3fO)

### CODE REVIEW

File: resources/views/components/layout.blade.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://cdn.tailwindcss.com?plugins=forms"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+
+    <title>Flight Planner</title>
+</head>
+
+<x-navbar />
```
Comment (ezef) [inline resources/views/components/layout.blade.php]: any HTML element must be inside `<body>` tag. Currently this layout is considered malformed. ( browsers fix them on the fly, that's why you dont see it)

File: resources/views/cities.blade.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                <th scope="col" class="py-3 px-6">ID</th>
+                <th scope="col" class="py-3 px-6">City</th>
+                <th scope="col" class="py-3 px-6">Country</th>
+                <th scope="col" class="py-3 px-6">Incoming Flights</th>
+                <th scope="col" class="py-3 px-6">Outgoing Flights</th>
+            </tr>
+            </thead>
+            <tbody>
+                <tr class="bg-white border-b border-gray-100">
+                    <form method="POST" action="cities/create">
```
Comment (guillemartinicore) [inline resources/views/cities.blade.php]: take a look at this. Challenge requires you to specifically make requests using Ajax.

File: resources/views/cities.blade.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<x-layout>
```
Comment (guillemartinicore) [inline resources/views/cities.blade.php]: don't you need some props here?

File: resources/views/cities.blade.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                        </td>
+                    </form>
+                </tr>
+                @foreach($cities as $city)
+                    <tr class="bg-white border-b border-gray-100">
+                        <td class="py-4 px-6">{{ $city->id }}</td>
+                        <td class="py-4 px-6">{{ $city->name }}</td>
+                        <td class="py-4 px-6">{{ $city->country }}</td>
+                        <td class="py-4 px-6">{{ $city->flightsTo->count() }}</td>
+                        <td class="py-4 px-6">{{ $city->flightsFrom->count() }}</td>
```
Comment (guillemartinicore) [inline resources/views/cities.blade.php]: Try not to do logic in your front-end. you can count them when making the database query. You can take a look at this https://laravel.com/docs/10.x/eloquent-relationships#counting-related-models

**Pull request merged**